examples/island_cluster_with_vm_router/vars.tf

```diff
@@ -0,0 +1,48 @@
+variable "project_id" {
+  type = string
+}
+
+variable "region" {
+  type = string
```

Comment:
Let's provide sane defaults (here and throughout) when possible.

Reply:
Most customers change the values, for this reason I kept all the values in `terraform.tfvars`.